docs/leaderboard/develop/api-reference/index.md

```diff
@@ -0,0 +1,193 @@
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+    * values(): {`id`: number, `score`: number, `rank`: number}[]
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+<SdkExampleTabs snippetId={'API_LeaderboardFetchByRank'} />
+
+For large leaderboards, you will need to fetch in batches of 8192 elements.
+
+<SdkExampleTabs snippetId={'API_LeaderboardFetchByRankPagination'} />
+
+### Fetch elements' rank
+
+Fetches elements given a list of element IDs. 
+
+
+| Name         | Type                                | Description                                                  |
+|--------------|-------------------------------------|--------------------------------------------------------------|
+| ids          | Array<`number`>                       | The ids of the elements whose rank we are retrieving.        |
+| order        | Optional[Ascending / Descending]    | The order to fetch the elements in. Defaults to ascending, meaning 0 is the lowest-scoring rank.   |
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+    * values(): {`id`: number, `score`: number, `rank`: number}[]
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+<SdkExampleTabs snippetId={'API_LeaderboardFetchByRank'} />
+
+For large leaderboards, you will need to fetch in batches of 8192 elements.
+
+<SdkExampleTabs snippetId={'API_LeaderboardFetchByRankPagination'} />
+
+### Get leaderboard length
+
+Gets the length of the leaderboard.
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+    * length(): number
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+<SdkExampleTabs snippetId={'API_LeaderboardLength'} />
+
+### Remove elements
+
+Removes elements with the specified IDs.
+
+| Name             | Type   | Description                                                                                                                                            |
+|------------------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ids        | Array<`number`> | An Array of element IDs for the elements you want to remove.                                                                                                                              |
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+<SdkExampleTabs snippetId={'API_LeaderboardRemoveElements'} />
+
+For large leaderboards, you will need to remove in batches of 8192 elements.
+
+<SdkExampleTabs snippetId={'API_LeaderboardRemoveElementsPagination'} />
+
+### Delete leaderboard
+
+Deletes the leaderboard.
```

Comment:
There aren't any parameters for this one? You might explicitly state that the leaderboard identifier is inferred from the leaderboard client.